src/components/editor-page/editor-pane/editor-pane.tsx

```diff
@@ -103,6 +98,29 @@ export const EditorPane: React.FC<EditorPaneProps & ScrollProps> = ({ onContentC
   const [editorScroll, setEditorScroll] = useState<ScrollInfo>()
   const onEditorScroll = useCallback((editor: Editor, data: ScrollInfo) => setEditorScroll(data), [])
 
+  const onPaste = useCallback((pasteEditor: Editor, event: PasteEvent) => {
+    if (!event || !event.clipboardData) {
+      return
+    }
+
+    // Check for pasted tables
+    const pasteText = event.clipboardData.getData('text')
+    if (smartPasteEnabled && pasteText && isTable(pasteText)) {
+      event.preventDefault()
+      extractTable(pasteText, pasteEditor)
+      return
+    }
```

Comment:
https://user-images.githubusercontent.com/6124140/107889213-4a809500-6f11-11eb-905f-c5f350f87e5c.mp4

Reply:
What's your idea how to detect whether the cursor is inside a code-fence or not?

Reply:
Check the line above and below (except blank lines)?

Reply:
I now did the following: Check if either the cursor's line, the one above or the one below have the `m-markdown comment` CodeMirro token, which is applied to codefence blocks.
I need to check the line above and below too as empty lines - although being inside the fence - have no CodeMirror token.

From this results the following:
 1. Reformatting won't trigger if the line above or below starts with `` ``` `` or is inside the fence and has any content.
 2. Reformatting will trigger if you're outside the codefence but directly above the fence-start (next line begins with `` ``` ``).
 3. Same as 2. when you're outside a codefence but the previous line is the end of it.